src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
+  const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
+
+  if (!response.ok) {
+    throw new Error(`${response.status} - ${response.statusText}`);
+  }
+
+  return response.json();
+}
+
+export async function delComment(id : number) {
```

Comment:
I would call it `deleteComment`. Not a big deal as for me, but [style-guyde](https://mate-academy.github.io/style-guides/javascript.html#2311-dont-shorten-variables-or-functions-names)